py3status/modules/mega.py

```diff
@@ -47,7 +48,7 @@ def mega(self):
         if len(output) == 0:
             self.py3.error(STRING_NOT_CONFIGURED)
 
-        columns = output[0].split()
+        columns = [col.lower() for col in output[0].split()]
```

Comment:
`columns = output[0].lower().split()` is better.

Reply:
Indeed